src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -1379,3 +1426,18 @@ TEST_CASE("FlattenTSwishFunctionTest", "[ActivationFunctionsTest]")
   CheckFlattenTSwishActivationCorrect(input, desiredActivation);
   CheckFlattenTSwishDerivateCorrect(desiredActivation, desiredDerivation);
 }
+
+/**
+ * Basic test of the Square NonLinearity (SQNL) function.
+ */
+TEST_CASE("SQNLFunctionTest", "[ActivationFunctionsTest]")
+{
+  // Calculated using custom functions
```

Comment:
What does custom function mean in this case?

Reply:
@zoq the cusom function is basically this: https://github.com/mlpack/mlpack/pull/3044#issuecomment-908478248